openwisp_controller/config/static/config/js/relevant_templates.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+django.jQuery(function ($) {
+    var selectedTemplates = django._owcInitialValues["config-0-templates"].split(','),
```

Comment:
Getting `selectedTemplates` using jQuery selector was resulting in "unsaved changes" dialog box appearing even when no changes were made due to order of template IDs. Using this here, resolved that issue.